nf_core/pipeline-template/{{cookiecutter.name_noslash}}/lib/Checks.groovy

```diff
@@ -0,0 +1,158 @@
+                 "  Auto-activating '--skip_biotype_qc' parameter to circumvent the issue below:\n" +
+                 "  https://github.com/nf-core/rnaseq/issues/460.\n\n" +
+                 "  If you would like to use the soft-masked Ensembl assembly instead please see:\n" +
+                 "  https://github.com/nf-core/rnaseq/issues/159#issuecomment-501184312\n" +
+                 "==================================================================================="
+    }
+
+    // Print a warning if using a UCSC assembly from igenomes.config
+    static void ucsc_genome_warn(log) {
+        log.warn "=============================================================================\n" +
+                 "  When using UCSC assemblies the 'gene_biotype' field is absent from the GTF file.\n" +
+                 "  Auto-activating '--skip_biotype_qc' parameter to circumvent the issue below:\n" +
+                 "  https://github.com/nf-core/rnaseq/issues/460.\n\n" +
+                 "  If you would like to use the soft-masked Ensembl assembly instead please see:\n" +
+                 "  https://github.com/nf-core/rnaseq/issues/159#issuecomment-501184312\n" +
+                 "==================================================================================="
+    }
+
+    // Print a warning if both GTF and GFF have been provided
+    static void gtf_gff_warn(log) {
+        log.warn "=============================================================================\n" +
+                 "  Both '--gtf' and '--gff' parameters have been provided.\n" +
+                 "  Using GTF file as priority.\n" +
+                 "==================================================================================="
+    }
+
+    // Print a warning if --skip_alignment has been provided
+    static void skip_alignment_warn(log) {
+        log.warn "=============================================================================\n" +
+                 "  '--skip_alignment' parameter has been provided.\n" +
+                 "  Skipping alignment, quantification and all downstream QC processes.\n" +
+                 "==================================================================================="
+    }
+
+    // Print a warning if using '--aligner star_rsem' and '--with_umi'
+    static void rsem_umi_error(log) {
+        log.error "=============================================================================\n" +
+                  "  When using '--aligner star_rsem', STAR is run by RSEM itself and so it is\n" +
+                  "  not possible to remove UMIs before the quantification.\n\n" +
+                  "  If you would like to remove UMI barcodes using the '--with_umi' option\n" + 
+                  "  please use either '--aligner star' or '--aligner hisat2'.\n" +
+                  "============================================================================="
+    }
```

Comment:
These functions all seem to be specific to the `rnaseq` pipeline. I don't think that they should be in the template really, as they are not at all relevant for many pipelines.